### DISCUSSION TITLE
`dotnet test -t` and fully qualified name

### DISCUSSION THREAD

**Discussion by WereDev** [nunit/nunit3-vs-adapter#897] [Q&A] [answered]
Using the dotnet cli, is there any way to get it to output the fully qualified name of tests when running `dotnet test -t`?

I ask because A) I think this would be useful and B), the VSCode extension .Net Core Test Explorer seems to require this in order to list the known tests.

Thanks!

**Comment by OsirisTerje** [comment, marked as answer]
You can try:
```cmd
dotnet test -t -- NUnit.DisplayName=FullName
```

Example:
![image](https://user-images.githubusercontent.com/203432/136714110-2b02045d-5060-4422-a090-f78623a756e2.png)

Can you point me to where it states that the VSCode extension .Net Core Test Explorer requires this?  That documentation should be updated :-)
